### ISSUE TITLE
Typos/redundancies on page 4

### ISSUE THREAD

**Issue by chriskorosu** [siefkenj/IBLLinearAlgebra#81]
https://github.com/siefkenj/IBLLinearAlgebra/blob/ed10b23b4014a47223242217953eaf81cb6e36de/problemsets-mat223/modules/module1.tex#L340

Lines 340-342

```
Formalizing, for vectors $\vec u$, $\vec v$, $\vec w$ and scalars $\alpha$ and $\beta$
are scalars, the
following are laws are always satisfied:
```

should be 

```
Formalizing, for vectors $\vec u$, $\vec v$, $\vec w$ and scalars $\alpha$ and $\beta$, the
following are always satisfied:
```

**Comment by siefkenj** [comment]
Thanks @chriskorosu ! Can you make a Pull Request for this change? I would like to keep the word `laws` in the text.

**Comment by chriskorosu** [comment]
Done, I think (I'm new to Github)

**Comment by siefkenj** [comment]
Fixed by PR #82
